amundsen_application/static/js/components/common/ResourceListItem/styles.scss

```diff
@@ -23,13 +23,17 @@
     opacity: 1;
   }
 
+  .bookmark-icon {
```

Comment:
If this isn't a merge conflict and affects the search results UI, this should be done on master or we will have conflicting work.